File: testcases/repeat_times.js
var i = 0
repeat_times(5) {
  print(i)
  print("hello")
  i = i + 1

  var j = 0
  repeat_times(5) {
    print(j * 1000)
    print("world")
    j = j + 1
  }
}

File: testcases/repeat_times.lua
local i = 0

local iter_0= 0;
while(iter_0<5)
do
  print(i)
  print("hello")
  i = (i + 1)
  local j = 0
  
  local iter_1= 0;
  while(iter_1<5)
  do
    print((j * 1000))
    print("world")
    j = (j + 1)
    iter_1=iter_1+1
  end
  iter_0=iter_0+1
end
